preprocessing/tabular_data/training_model_on_tabular_data.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p><strong>Rephrase: </strong></p><p>"This notebook is a sequel to the <a href="https://github.com/awslabs/amazon-sagemaker-examples/raw/ee21044e47bebab98b7853e3dd7557abfe2d3467/preprocessing/tabular_data/preprocessing_tabular_data.ipynb" target="_blank">preprocessing_tabular_data.ipynb</a> notebook and assumes that your tabular data is preprocessed as shown in it. That is, before we get started make sure to run preprocessing_tabular_data.ipynb before running this notebook. "</p><p> </p><p>This is a bit awkward. Can you rephrase this? Suggestion:</p><p> </p><blockquote>"This notebook is a sequel to the <a href="https://github.com/awslabs/amazon-sagemaker-examples/raw/ee21044e47bebab98b7853e3dd7557abfe2d3467/preprocessing/tabular_data/preprocessing_tabular_data.ipynb" target="_blank">preprocessing_tabular_data.ipynb</a> notebook. Before running this notebook, run preprocessing_tabular_data.ipynb to preprocess the data used in this notebook."</blockquote><p> </p><p><strong>Question: </strong></p><p>Do users need to run both notebooks within a specific time period for this to work - how long will variables persist when using storemagic?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>